.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+labels: 'feature'
+
+---
+
+## Background
+A clear and concise description of where the limitation lies.
+
+## Feature
+A description of the requested feature.
+
+## Example [Optional]
+A simple example if applicable.
+
+## Proposed Solution [Optional]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)